### PR TITLE
[enhancement](regression-test) Add regression test for presto dialect

### PR DESCRIPTION
**Pull request by suxiaogang223**
## Proposed changes
Add regression test for presto dialect

### CODE REVIEW

File: regression-test/suites/dialect_compatible/presto/date.sql [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+-- SELECT dow(DATE '2020-05-10');
+-- SELECT dow(DATE '1960-05-10');
+-- SELECT EXTRACT(DAY_OF_YEAR FROM DATE '2020-05-10');
+-- SELECT EXTRACT(DAY_OF_YEAR FROM DATE '1960-05-10');
+-- SELECT day_of_year(DATE '2020-05-10');
+-- SELECT day_of_year(DATE '1960-05-10');
+-- SELECT EXTRACT(DOY FROM DATE '2020-05-10');
+-- SELECT EXTRACT(DOY FROM DATE '1960-05-10');
+-- SELECT doy(DATE '2020-05-10');
+-- SELECT doy(DATE '1960-05-10');
```
Comment (superdiaodiao) [inline regression-test/suites/dialect_compatible/presto/date.sql:39]: If we don't need these queries, maybe remove them would be better?

**Comment by suxiaogang223** [comment]
run buildall

File: regression-test/suites/dialect_compatible/presto/date.sql (hunk 1 above)
Comment (morningman) [reply regression-test/suites/dialect_compatible/presto/date.sql]: this will be fixed in future

**Comment by suxiaogang223** [comment]
run buildall

**Comment by suxiaogang223** [comment]
run buildall

**Comment by morningman** [comment]
run buildall

**Review by wuwenchi — APPROVED** [review]
LGTM

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**